public/exercism_completion.bash

```diff
@@ -1,7 +1,71 @@
-__from_exercism_config() {
-  COMPREPLY=()
-  local cur=${COMP_WORDS[COMP_CWORD]}
-  COMPREPLY=($(compgen -W 'demo debug configure fetch restore submit unsubmit tracks download help' -- $cur))
+_exercism () {
+  local cur prev
+
+  COMPREPLY=()   # Array variable storing the possible completions.
+  cur=${COMP_WORDS[COMP_CWORD]}
+  prev=${COMP_WORDS[COMP_CWORD-1]}
+
+  commands="configure debug download fetch list open
+  restore skip status submit tracks unsubmit
+  upgrade help"
+  tracks="csharp cpp clojure coffeescript lisp crystal
```

Comment:
Yeah. Maybe the answer is to add something to the deploy script to update this repo if the list of tracks changes. Then auto-update and auto-deploy. That actually wouldn't be too hard since I already have a script doing deploys.